src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+
+  --blue-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
```

Comment:
Avoid fixing container sizes unless required. Consider using relative units or allowing content to dictate the size to prevent overflow or accidental scroll bars.